solutions/kubernetes/cluster-functions.sh

```diff
@@ -317,18 +309,11 @@ function setup_primary_node(){
 }
 
 function join_worker_nodes() {
-    echo "---------------------------------------[ Joining Worker Node $HOSTNAME ]--------------------------------------"
+    add_secondary_separator "Joining Worker Node $HOSTNAME"
     echo 'y' | kubeadm reset && "${@:2}"
     check_status "Failed to join $HOSTNAME node to cluster"
 }
 
-ACTION="$1"
```

Comment:
why we removed this?

Reply:
Didn't remove it relocated at top at line 30 as it's the first thing script requires.